SSI.php

```diff
@@ -25,9 +25,6 @@
 global $smcFunc, $ssi_db_user, $scripturl, $ssi_db_passwd, $db_passwd, $cachedir;
 
 // Remember the current configuration so it can be set back.
-$ssi_magic_quotes_runtime = function_exists('get_magic_quotes_gpc') && get_magic_quotes_runtime();
-if (function_exists('set_magic_quotes_runtime'))
-	@set_magic_quotes_runtime(0);
```

Comment:
According to install.php and upgrade.php, our minimum is still PHP 5.3.8. If there has been a recent change that is compatible only with 5.4+, we need either to revert that change or to deliberate about whether to raise our minimum PHP level. In the meantime, we need to leave the magic quotes stuff in place.